.travis-before-script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+# Add needed dependencies.
+cd "$DRUPAL_TI_DRUPAL_DIR"
+ln -sf $(pwd) subdir
```

Comment:
Uhm, probably it would make more sense to use a .travis-after-script.sh and then have this run after drupal installation, but as "drupal_ti_ensure_drupal" is re-rentrant, it does not matter much either way.